ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -39,6 +49,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: operator-unit-test
+  steps:
+    test:
+    - as: unit
+      commands: make test
+      credentials:
+      - mount_path: /var/run/oadp-credentials
+        name: oadp-credentials
+        namespace: test-credentials
+      from: test
```

Comment:
This tries to use an unknown name for the image - change this to `test-oadp-operator`.